src/axom/quest/examples/CMakeLists.txt

```diff
@@ -32,6 +33,115 @@ if(AXOM_ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
         DEPENDS_ON  ${quest_example_depends} mfem
         FOLDER      axom/quest/examples
         )
+
+    if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR)
```

Comment:
These are new unit tests that use the new inline mesh w/ the shaping driver.
They're all sampling-based, but we should add some intersection-based tests in a follow-up PR.